src/app/(privateRoutes)/dashboard/viewpdf/page.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
```

Comment:
Is this import necessary?
if it is not eliminated

Reply:
remove